angularjs-portal-frame/pom.xml

```diff
@@ -38,33 +42,46 @@
         <artifactId>lesscss-maven-plugin</artifactId>
         <version>1.7.0.1.1</version>
         <configuration>
-            <sourceDirectory>${project.basedir}/src/main/webapp/css/</sourceDirectory>
-            <outputDirectory>${project.build.directory}/${project.build.finalName}/css/</outputDirectory>
-            <compress>true</compress>
-            <includes>
-                <include>angular.less</include>
-            </includes>
-            <outputFileFormat>{fileName}.${project.version}.css</outputFileFormat>
+          <sourceDirectory>${project.basedir}/src/main/webapp/css/</sourceDirectory>
+          <outputDirectory>${webapp.target.directory}/css/</outputDirectory>
+          <compress>true</compress>
+          <includes>
+            <include>angular.less</include>
+          </includes>
+          <outputFileFormat>{fileName}.${project.version}.css</outputFileFormat>
         </configuration>
         <executions>
-            <execution>
-                <goals>
-                    <goal>compile</goal>
-                </goals>
-            </execution>
+          <execution>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+          </execution>
         </executions>
       </plugin>
       <plugin>
-          <groupId>org.apache.tomcat.maven</groupId>
-          <artifactId>tomcat7-maven-plugin</artifactId>
-          <version>2.2</version>
-          <configuration>
-            <url>http://localhost:8080/manager/text</url>
-            <server>TomcatServer</server>
-            <path>/frame</path>
-          </configuration>
-        </plugin>
+        <groupId>org.apache.tomcat.maven</groupId>
+        <artifactId>tomcat7-maven-plugin</artifactId>
+        <version>2.2</version>
+        <configuration>
+          <url>http://localhost:8080/manager/text</url>
+          <server>TomcatServer</server>
+          <path>/frame</path>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>9.3.0.M2</version>
+        <configuration>
+          <webApp>
+            <baseResource implementation="org.eclipse.jetty.util.resource.ResourceCollection">
+              <resourcesAsCSV>src/main/webapp,${webapp.target.directory}</resourcesAsCSV>
```

Comment:
Yeah, I never even realized jetty:run looks at source rather than target.  Maybe `mvn jetty:run-war` is the sensible thing to do, but this is faster.